frontend/public/components/monitoring/dashboards/table.tsx

```diff
@@ -122,7 +122,11 @@ const Table: React.FC<Props> = ({ panel, pollInterval, queries, namespace }) =>
     return <ErrorAlert message={t('public~panel.styles attribute not found')} />;
   }
   if (_.isEmpty(data)) {
-    return <EmptyBox label={t('public~Data')} />;
+    return (
+      <div className="cos-status-box">
```

Comment:
I worry about depending on a console style from a (future) plugin since it's not an API. Is there a PF style we can use here and potentially for `EmptyBox` as well? Maybe extra small `EmptyState`?

https://patternfly-react-main.surge.sh/components/empty-state#extra-small

cc @rhamilto @sg00dwin

Reply:
```
  <EmptyState variant={EmptyStateVariant.xs}>
    <EmptyStateBody>
      No data
    </EmptyStateBody>
  </EmptyState>
```

results in 

<img width="873" alt="Screenshot 2022-10-31 at 8 31 22 AM" src="https://user-images.githubusercontent.com/895728/199008338-19fbaa87-24c8-420e-90a8-ecc549324474.png">

Note the text is gray instead of black like it was.  

To avoid fragmentation, I think it's probably best to leave `EmptyBox` in place and for the console team to generate a story to revisit all the components in https://github.com/openshift/console/blob/master/frontend/public/components/utils/status-box.tsx.  Most of these are from the CoreOS days.  Thoughts?

Reply:
Thank you @spadgett, @rhamilto!

I updated this PR to use `EmptyState` like you suggested.